tests/test_e2e.py

```
@@ -460,43 +583,93 @@ def test_in_list():
+    | text_st
+    | st.lists(st.booleans() | st.integers(-10, 10) | text_st)
+    | st.dictionaries(
+        at_least_1_text_st, st.booleans() | st.integers(-10, 10) | text_st
+    ),
+)
 def test_prop_equal2(a, b):
     res = query("RETURN $a = $b", params={"a": a, "b": b})
     if isinstance(a, list) and isinstance(b, list):
-        assert res.result_set == [[a == b and all(type(x) == type(y) for x, y in zip(a, b))]]
+        assert res.result_set == [
+            [a == b and all(type(x) == type(y) for x, y in zip(a, b))]
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)